calculator/src/main/java/edu/nextstep/camp/calculator/ViewAdapter.kt

```diff
@@ -0,0 +1,28 @@
+package edu.nextstep.camp.calculator
+
+import android.view.View
+import android.widget.Button
+import android.widget.TextView
+import androidx.databinding.BindingAdapter
+import androidx.recyclerview.widget.RecyclerView
+
+@BindingAdapter(value = ["setEnabledByViewType"])
+fun Button.setEnabledByViewType(viewType: CalculatorViewType?) {
+    this.isEnabled = if (viewType == null) true else viewType is ExpressionView
```

Comment:
```suggestion
    if (viewType == null) return 
    this.isEnabled = viewType is ExpressionView
```
isEnabled의 디폴트값은 true이기 때문에, 이렇게 작성하시면 좀 더 보기 좋을 것 같아요.